src/Configuration.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace Wheniwork\Feedback;
+
+use Auryn\Injector;
+
+class Configuration
+{
+    public function apply(Injector $injector, array $env)
+    {
+        $injector->define("Predis\Client", []);
+        $injector->define("GuzzleHttp\Client", []);
```

Comment:
I think these are unneeded. Unless the constructor has a non-optional array parameter, which I don't believe is the case for either of these classes, the injector should handle this automatically.